pkgs/development/compilers/circt/circt-llvm.nix

```
@@ -0,0 +1,57 @@
+{ stdenv
+, cmake
+, ninja
+, circt
+, llvm
+, python3
+}: stdenv.mkDerivation {
+  pname = circt.pname + "-llvm";
+  version = circt.version;
+  src = circt.src + /llvm;
```

```suggestion
  inherit (circt) version src;
```

since `src = circt.src + /llvm;` unnecessarily copies the source.

---

Less copies sounds good. I'll fix it.